OKTA-479360 Update SCIM Provisioning integration images (#3154)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/scim-provisioning-integration-connect/main/index.md

### To Okta

This page contains settings for all information that flows from your application upstream into Okta.

Click **Edit** to make changes to the following sections.

* **General**: Schedules imports and dictates a username format for imported users. You can also define a percentage of acceptable assignments before the [import safeguards](https://help.okta.com/okta_help.htm?id=csh-eu-import-safeguard) feature is automatically triggered.

    If the Okta username is overridden due to mapping from a provisioning-enabled app, the custom mapping appears in this section.

* **User Creation & Matching**: Sets up matching rules to use when importing users from your application. Establishing matching criteria allows you to specify how an imported user is defined when creating a new user or mapping to an existing Okta user.

>   * **Imported user is an exact match to Okta user if**: Sets the matching criteria to determine when an imported user matches to an existing Okta user. For the new imported user to be considered an exact match, each option that you select must be true. Note that if you choose the third option, the first and second choices are disabled.

Reviewer: ```suggestion
  * **Imported user is an exact match to Okta user if**: Sets the matching criteria to determine when an imported user matches to an existing Okta user. To consider the new imported user an exact match, each option that you select must be true. Note that if you choose the third option, the first and second choices are disabled.
```

Before you can delete an attribute, you first need to remove the mapping for that attribute.

  1. From the Admin Console, open your SCIM integration.

  1. Go to the **Provisioning** tab. Under the **Settings** section, click **To App**.

      ![Displays the Provisioning > To App tab.](/img/oin/scim_check-attributes-1.png "Provisioning to App tab")

  1. Scroll to the **Attribute Mappings** section. Look for the attribute that you want to delete and then click the corresponding **X**.

      ![Displays the Attribute Mappings screen.](/img/oin/scim_check-attributes-2.png "Attribute Mappings")

>   1. Click **OK** to confirm that you want to remove the mapping for the attribute you selected.

Reviewer: ```suggestion
  1. Click **OK** to confirm that you want to remove the mapping for the attribute that you selected.
```

        Repeat steps 3 and 4 until you remove all the mappings for the attributes that you want to delete.

  1. After removing all the mappings for the attributes that you want to delete, click **Go to Profile Editor** in the **Attribute Mappings** section.

  1. In the Profile Editor, look for the attribute that you want to delete, and click the corresponding **X**.

        ![Displays the Profile Editor > Remove Attribute screen.](/img/oin/scim_check-attributes-7.png "Profile Editor: Remove Attribute")

  1. Click **Delete Attribute** to confirm that you want to remove the attribute.

        Repeat steps 6 and 7 for all the attributes that you want to delete.

### Add attributes

1. From the Admin Console, open your SCIM integration.

1. Go to the **Provisioning** tab. Under the **Settings** section, click **To App**.

1. Scroll to the **Attribute Mappings** section. Click **Go to Profile Editor**.

1. In the Profile Editor, click **Add Attribute**.

1. Enter the information for the new attribute that you’re adding and then click **Save**. For example:

    ![Displays the Add Attribute dialog](/img/oin/scim_check-attributes-11.png "Profile Editor: Add Attribute")

>     > **Note:** The Scope property determines whether the attribute you are adding can be assigned at a group level or just per user. If you would like your admins to be able to assign a value for this attribute at a group level, don't check the **User personal** checkbox.

Reviewer: should Scope be bolded or `Scope`?
```suggestion
    > **Note:** The Scope property determines whether the attribute that you are adding can be assigned at a group level or just per user. If you want your admins to be able to assign a value for this attribute at a group level, don't select the **User personal** checkbox.
```

Author: Yes, Scope needs to be bolded. Thanks!

1. After adding an attribute, you can add a mapping for that new attribute.

### Map attributes

1. From the Admin Console, open your SCIM integration.

1. Go to the **Provisioning** tab. Under the **Settings** section, click **To App**.

1. Scroll to the **Attribute Mappings** section. Look for the attribute that you want to update and click **Edit**.

    ![Displays the Attribute Mappings > Edit Attribute screen.](/img/oin/scim_check-attributes-13.png "Attribute Mappings: Edit Attribute")

> 1. In the dialog that appears, there are two dropdown fields. In the first dropdown menu, select **Map from Okta Profile**. In the second dropdown menu, choose the Okta profile attribute that you would like to map the SCIM attribute from. Click **Save**.

Reviewer: ```suggestion
1. In the dialog that appears, there are two dropdown fields. In the first dropdown menu, select **Map from Okta Profile**. In the second dropdown menu, choose the Okta profile attribute that you want to map the SCIM attribute from. Click **Save**.
```

    ![Displays the Map Attribute dialog.](/img/oin/scim_check-attributes-14.png "Attributes: Map Attribute")

> 1. Repeat steps 3 and 4 for all other SCIM attributes you want to map (from Okta to your application).

Reviewer: ```suggestion
1. Repeat steps 3 and 4 for all other SCIM attributes that you want to map (from Okta to your application).
```

> 1. After updating the mappings from Okta to your application, click **To Okta** under the **Settings** section.

Reviewer: ```suggestion
1. After you update the mappings from Okta to your application, click **To Okta** under the **Settings** section.
```

1. If your integration supports User Imports, try importing one user from your application. Check the imported user and ensure that the values for supported attributes are reflected in that imported user's account in Okta.

    1. From your Admin Console, select **Directory** > **People**. The list of Okta users for your org appears.

    1. Find the user that you imported and click the user's name.

>     1. Once the user account appears, click **Profile**. The Profile screen shows the user's attributes. Verify that the supported attribute values were imported properly.

Reviewer: ```suggestion
    1. After the user account appears, click **Profile**. The Profile screen shows the user's attributes. Verify that the supported attribute values were imported properly.
```
